source/parameters/aperture.md

```diff
@@ -111,9 +113,10 @@ Between vertex points, the aperture can follow a straight line or the arc of an
 The vertex points are specified by setting the `vertices` parameter. This parameter has three
 subcomponents
 ```{code} yaml
-center              # [m] Vector of [x, y] center point.
-absolute_vertices   # Boolian. Default is False.
-list                # Struct. Ordered list of vertex points.
+  verticies:
+    center: 0                 # [m] Vector of (x, y) center point.
+    absolute_vertices: false  # [Boolean] Default is False.
+    list: [{}]                # [List of dictionaries] Ordered list of vertex points.
```

Comment:
How about `[Array]` instead of `[List of dictionaries]`?

Reply:
Both are ok wordings in that context, we just need to use it consistently throughout the document.

[YAML calls it](https://yaml.org/spec/1.2.2/) `sequences (arrays/lists)` in its specs. A Python `list` is what one gets when reading it in (not a continuous array data structure).

Reply:
Also remember that the standard is language agnostic. The reason I like the name `array` over `list` is that `array` implies that all the elements are of the same type.